arcsi/handler/upload.py

```diff
@@ -160,6 +160,9 @@ def upload(self,):
                         remaining_byte -= chunk_byte
                         start += chunk_byte
                         app.logger.debug("chunk posted! {} bytes remaining".format(remaining_byte))
+                    else: 
+                        app.logger.debug("Azuracast response error code: {}, body: {}".format(req.status_code, req.text))
+                        return False #exit while loop when error
```

Comment:
Do we expect here that after exit the requestexception is raised and new round of retry happens?

Reply:
No. The finding here was exactly that if a file cannot be uploaded due to tricky file name, then there will be no request exception, only the request will not be "ok" (false value). A retry would make no sense it this case because Azura will deterministically reject the file. 

But the exception case is good to have, I think. So it's good you added it. I'm not sure I can think of an easy case where it helps, but we never know.

Reply:
What's the Az response like? status code is 200 and there's json payload w/ the message request? Yeah there is no point in the retry mechanism for rejected uploads. That only is supposed to help w/ network connection is an issue. We can later remove it or rewrite this a bit to use the json payload to catch exceptions on upload like this. 

But let's go ahead w/ this now I think. Thanks for the details these are extremely useful to know

Reply:
> What's the Az response like? status code is 200 and there's json payload w/ the message request?

Like this: https://github.com/mmmnmnm/lahmacun_arcsi/issues/54#issuecomment-761234391